### DISCUSSION TITLE
مشکل dns over https

### DISCUSSION THREAD

**Discussion by Evolve6996** [hiddify/Hiddify-Manager#3284] [Q&A سوالات رایج]
سلام ممنون برای پنل فوق العادتون 
فقط نمیدونم چرا دی ان اس کار نمیکنه. 

![Untitled](https://github.com/hiddify/Hiddify-Server/assets/91710620/3be5f6c6-0c32-43f2-a3be-03cdff2a39a8)

**Comment by raminrez** [comment]
@lymanjre @hiddify-com  @HiddifyOfficial 
سلام دوستان
درمورد استفاده از DOH راهنمایی یا داکیومنتی دارید؟ منم نتونستم ازش استفاده کنم
